.github/workflows/benchmark.yml

```
@@ -3,7 +3,7 @@ name: Benchmark
 on:
   pull_request_target:
     branches: [ main ]
-    types: [ opened ]
+    types: [ opened, synchronize ]
```

Remove testing action trigger
```suggestion
    types: [ opened ]
```